src/app/shared/map/map.component.ts

```diff
@@ -80,6 +85,25 @@ export class MapComponent {
 				this.placedMarker = event.coordinate;
 			});
 		}
+
+		this.map.getView().on('change:resolution', (event) => {
+			const zoom = this.map?.getView().getZoom();
+			const center = this.map?.getView().getCenter();
+			// Ensure that zoom and center change events are only emitted when they actually changed to prevent an event flood
+			if (zoom && Math.abs(zoom - this.lastZoomEvent) > 0.5) {
+				this.zoomChange.emit(zoom);
+				this.lastZoomEvent = zoom;
+			}
+
+			if (
+				center &&
+				Math.abs(center[0] - this.lastCenterEvent[0]) > 0.1 &&
+				Math.abs(center[1] - this.lastCenterEvent[1]) > 0.1
+			) {
+				this.centerChange.emit(center);
+				this.lastCenterEvent = center;
+			}
+		});
 	}
 
 	public refreshMap() {
```

Comment:
refresh is still not working fully, If I solve a location riddle I have to refresh the app to see the guesses and solution of others. To reproduce log in with the demo user and try to solve a riddle (use the initial setup which is created when using docker compose) or see backend/infrastructure/default_state_setup/README

Reply:
Fix: https://github.com/uzh-ase-fs24/frontend/pull/44/commits/17f4887b1be363e779803076b74bdda161151723